apps/root/src/pages/strategy-guardian-detail/strategy-management/withdraw/delayed-withdraw-container/index.tsx

```diff
@@ -124,14 +130,31 @@ const DelayedWithdrawItem = ({ delayed, type, position }: DelayedWithdrawItemPro
           </ContainerBox>
         </ContainerBox>
       </ContainerBox>
-      {type === DelayedWithdrawalStatus.READY && (
-        <Button variant="outlined">
-          <FormattedMessage
-            description="earn.strategy-management.withdraw.delayed-withdraw.item.withdraw-now"
-            defaultMessage="Withdraw now"
-          />
-        </Button>
-      )}
+      {type === DelayedWithdrawalStatus.READY &&
+        (!!position.pendingTransaction ? (
```

Comment:
No need for this since they can submit several delayed withdraws at the same time

Reply:
This would only disable that particular button, so they don't submit the same transaction, and they can check status

Reply:
Ah right since we have only 1 pending and 1 ready by position